src/stimfit/gui/graph.cpp

```diff
@@ -2175,7 +2175,7 @@ void wxStfGraph::ChanScroll(int direction) {
     int last_chan = Doc()->size()-1;
     
     // Exit early if there is only one channel    
-    if (last_chan == 0) {
+    if (Doc()->size() == 1) {
```

Comment:
- The no file case does nothing, unless the python terminal is selected, in which case it moves through the history.
- 1 channel : no effect
- 2 channel case : No effect
- 20 channels : works as expected